### ISSUE TITLE
Re-add removed part of test from "moves to another task when dragged over"

### ISSUE THREAD

**Issue by lisa-fehr** [ProcessMaker/modeler#724]
## Description
Start after #718 is merged

https://github.com/ProcessMaker/modeler/pull/718/files#diff-dd7181926d02b18e486a36f5ff941f91L242
There is a test with two tasks where you drag a boundary event from one task to the other, then drag one task and then the other. The position of the boundary event is calculated with `close to` and `not close to`. But it was off by about 62px left. It needs to be added as a new test, or the position mystery solved (maybe it included the missing label somehow before the highlighter change from box&label to shape).

#723 might help with this issue.
